src/util/log/fd_log.c

```diff
@@ -56,6 +56,10 @@
 #include <syscall.h>
 #include <execinfo.h>
 
+/* Log buffer, used by both fd_log_private_0 and fd_log_private_hexdump_msg */
+
+static FD_TLS char log_msg[ FD_LOG_BUF_SZ ];
```

Comment:
(Ultra pedantic ... probably would name this with a fd_log_private_log_msg to make symbol table dumps easier to read / grep / etc.)